components/gfx/paint_context.rs

```diff
@@ -466,6 +466,163 @@ impl<'a> PaintContext<'a> {
         }
     }
 
+    fn is_zero_radius(radius: &Size2D<AzFloat>) -> bool {
+        radius.width <= 0. || radius.height <= 0.
+    }
+
+    fn ellipse_to_bezier(path_builder: &mut PathBuilder,
+                         origin: Point2D<AzFloat>,
+                         radius: Size2D<AzFloat>,
+                         start_angle: f32,
+                         end_angle: f32) {
+        if PaintContext::is_zero_radius(&radius) {
+            return;
+        }
+
+        // Calculate kappa constant for partial curve. The sign of angle in the
+        // tangent will actually ensure this is negative for a counter clockwise
+        // sweep, so changing signs later isn't needed.
```

Comment:
Do you have a reference where I can read about this? (Code you referenced from is OK.)